CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent changes
```

Comment:
Sorry, this should have been `## Recent Changes` with capitalized second word, will push additional commit in a moment.